src/events/SDL_events.c

```diff
@@ -731,16 +755,23 @@ SDL_PeepEvents(SDL_Event * events, int numevents, SDL_eventaction action,
     return (used);
 }
 
+int
+SDL_PeepEvents(SDL_Event * events, int numevents, SDL_eventaction action,
+               Uint32 minType, Uint32 maxType)
+{
+    return PeepEventsInternal(events, numevents, action, minType, maxType, PEEP_EVENTS_FLAGS_NONE);
+}
+
 SDL_bool
 SDL_HasEvent(Uint32 type)
 {
-    return (SDL_PeepEvents(NULL, 0, SDL_PEEKEVENT, type, type) > 0);
+    return (PeepEventsInternal(NULL, 0, SDL_PEEKEVENT, type, type, PEEP_EVENTS_FLAGS_NONE) > 0);
```

Comment:
I've made all the other callers of `SDL_PeepEvents` use `PeepEventsInternal` directly instead of going via `SDL_PeepEvents`, which might be a bit more efficient because it will bypass ELF interposing, but these could be reverted if desired.